torch/distributed/_composable/fsdp/_fsdp_collectives.py

```diff
@@ -185,8 +185,7 @@ def chunk_cat(
     simplefsdp: bool = False,
 ) -> torch.Tensor:
     if simplefsdp:
-        reduce_dtype = torch.bfloat16
-        tensors = [t.type(reduce_dtype) for t in tensors]
+        tensors = [t.type(torch.bfloat16) for t in tensors]
```

Comment:
why we are doing this conversion? I thought reduce dtype should normally be fp32.